OstelcoStyles/LoopingVideoView.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  LoopingVideoView.swift
+//  OstelcoStyles
+//
+//  Created by Ellen Shapiro on 5/22/19.
+//  Copyright © 2019 mac. All rights reserved.
+//
+
+import AVFoundation
+import UIKit
+
+/// A view designed to play looping videos to look like GIFs
+/// without the horrendous memory overhead of actual GIFs
+@IBDesignable
+public class LoopingVideoView: UIView {
+    
+    private lazy var gifPlayer = AVQueuePlayer()
+    private lazy var playerLayer: AVPlayerLayer = {
+        let playerLayer = AVPlayerLayer(player: self.gifPlayer)
+        playerLayer.videoGravity = .resizeAspect
+    
+        // Get rid of doofy gray border on player layer
+        playerLayer.shouldRasterize = true
+        playerLayer.rasterizationScale = UIScreen.main.scale
+        
+        return playerLayer
+    }()
+    private(set) var playerLooper: AVPlayerLooper?
+    
+    // Note: This label only shows in Interface Builder so you can tell where the view will go.
+    private lazy var ibPlaceholder: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.text = "Gif goes here!"
```

Comment:
Should this be an icon instead of label ? Something like this: https://www.flaticon.com/premium-icon/film_1146203

Reply:
Normally I'd be with you on this, but since this is only there for Interface Builder as a placeholder (and would never be shown to an actual user since it's *only* ever accessed in `prepareForInterfaceBuilder`, it'd add weight to the `.ipa` without actual benefit to the user. I think the text is sufficient as a placeholder for building things out in IB.